lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/gateway/KBVGateway.java

```diff
@@ -78,10 +87,14 @@ public QuestionsResponse getQuestions(QuestionRequest questionRequest) {
 
     @Tracing
     public QuestionsResponse submitAnswers(QuestionAnswerRequest questionAnswerRequest) {
-        TracingUtils.putAnnotation(EXPERIAN_IIQ_REQUEST, "rtq");
+        TracingUtils.putAnnotation(EXPERIAN_IIQ_REQUEST, EXPERIAN_RESPONSE_TO_QUESTIONS);
```

Comment:
If this putAnnotation method isn't serving any purpose at the moment then it can be removed